component/admin/views/webservices/tmpl/default.php

```diff
@@ -49,8 +50,12 @@ function setWebservice(client, webservice, version, folder, task)
 			var url = jQuery(this).attr('data-remote');
 			var format = jQuery(this).attr('data-remote-format');
 			var dataType = format == 'json' ? 'json' : 'text';
+			var ToC =
+				"<nav role='navigation' class='table-of-contents'>" +
```

Comment:
this should belong in the content and then you can just find the ul element and add li's to it...